frontend/packages/data-portal/app/components/Breadcrumbs.tsx

```diff
@@ -75,16 +75,24 @@ export function Breadcrumbs({
     return `${url}?${encodedParams}`
   }, [singleDatasetHistory, variant, data])
 
+  const returnToDepositionLink = useMemo(() => {
```

Comment:
nit - why does this need to be memoized?

Reply:
just a habit, I guess it's not a taxing computation so I can remove the memo